src/content/events/2024/bytesize_pipeline_container_URLs.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Bytesize: pipeline container URLs'
+subtitle: Phil Ewels, Seqera
+type: talk
+startDate: '2024-05-28'
+startTime: '13:00+00:00'
+endDate: '2024-05-28'
+endTime: '13:30+00:00'
+locations:
+  - https://kth-se.zoom.us/j/68390542812
```

Comment:
```suggestion
location:
  - links: https://kth-se.zoom.us/j/68390542812
```
new structure dropped during hackathon. readme update is stuck in a different PR 🙂